bundles/ui/org.openhab.ui.dashboard/src/main/java/org/openhab/ui/dashboard/DashboardTile.java

```diff
@@ -31,6 +31,13 @@
     String getUrl();
     
     /**
+     * The url to point to for the dashboard tile.
+     * (if it is a local UI, it should be a relative path starting with "../")
+     * @return the tile url
+     */
+    String getDashboardTile();
```

Comment:
May I suggest to rename this method to getImageUrl()? Having a DashboardTile.getDashboardTile() somehow sounds recursive to me ;-)
Apart from that: Thanks for the quick PR! I actually didn't think of such an easy solution, I was somehow wondering how to send the image streams around and serving them through the servlet. But solving this with simple urls is of course much better!